### ISSUE TITLE
Ventura and intel wifi support

### ISSUE THREAD

**Issue by cool2apps** [LeeBinder/Asus-Vivobook-S510UA-Hackintosh#15]
**What is your laptop model? Does it fit the specs in [README](https://github.com/whatnameisit/Asus-Vivobook-X510UA-BQ490-Catalina-10.15.6-Hackintosh/blob/master/README.md#system-specification)?**
S510UN

**Is your feature request related to a problem? Please describe.**
mac OS 13 Ventura and intel wifi support

**Describe the solution you'd like**
mac OS 13 Ventura and intel wifi supported

**Additional context**

**Comment by LeeBinder** [comment]
Hi cool2apps. I have Ventura running here. Might be able to update the repo on the weekend, but honestly can't promise (have been and am super-busy).

Re. Intel Wi-Fi support: I've swapped the Intel in my VivoBook with a very old but original Mac Wi-Fi card, but if you want to stick to the stock Intel Wi-Fi, you'd need to check out [itlwm](https://github.com/OpenIntelWireless/itlwm#itlwm). I can't say anything about it because I can't test it, but they write it's compatible with the Intel Dual Band Wireless-AC 8265, if that's the one in your S510UN.

**Comment by cool2apps** [comment]
> Hi cool2apps. I have Ventura running here. Might be able to update the repo on the weekend, but honestly can't promise (have been and am super-busy).
> 
> Re. Intel Wi-Fi support: I've swapped the Intel in my VivoBook with a very old but original Mac Wi-Fi card, but if you want to stick to the stock Intel Wi-Fi, you'd need to check out [itlwm](https://github.com/OpenIntelWireless/itlwm#itlwm). I can't say anything about it because I can't test it, but they write it's compatible with the Intel Dual Band Wireless-AC 8265, if that's the one in your S510UN.

Thanks for reply. After that, I was able to solve the wifi problem partially. Now stock intel wifi is working but it seems to be download speed is too slow. As my hackintosh is not my daily OS, i can live with that, I guess.

On the other hand, sound is not working after upgrading from BigSur to Monterey. And still does not work with Ventura. Do you have any idea to fix this too?

**Comment by LeeBinder** [comment]
Great you managed to make stock intel wifi working with itlwm.

Sound:
- update both bootloaders, not just OC (e.g. via the [HackinDROM App](https://www.tonymacx86.com/threads/hackindrom-app-for-opencore-efi-creation-and-update.312176/)) but also Clover, to their latest versions, + latest kexts
- open OC config.plist in the latest OpenCore Configurator and save it
- restart and test in both bootloaders, OC and Clover

Let me know. Can try to aim for this weekend but really can't promise.

Good luck 👍

**Comment by cool2apps** [comment]
> Great you managed to make stock intel wifi working with itlwm.
> 
> Sound:
> 
>     * update both bootloaders, not just OC (e.g. via the [HackinDROM App](https://www.tonymacx86.com/threads/hackindrom-app-for-opencore-efi-creation-and-update.312176/)) but also Clover, to their latest versions, + latest kexts
> 
>     * open OC config.plist in the latest OpenCore Configurator and save it
> 
>     * restart and test in both bootloaders, OC and Clover
> 
> 
> Let me know. Can try to aim for this weekend but really can't promise.
> 
> Good luck 👍

Thanks. I will try when I have some time.

**Comment by LeeBinder** [comment]
way too much going in. Aiming for next week-end now

**Comment by abhishekthomasv** [comment]
I'd be glad if you could update the repo.

> Hi cool2apps. I have Ventura running here. Might be able to update the repo on the weekend, but honestly can't promise (have been and am super-busy).
> 
> Re. Intel Wi-Fi support: I've swapped the Intel in my VivoBook with a very old but original Mac Wi-Fi card, but if you want to stick to the stock Intel Wi-Fi, you'd need to check out [itlwm](https://github.com/OpenIntelWireless/itlwm#itlwm). I can't say anything about it because I can't test it, but they write it's compatible with the Intel Dual Band Wireless-AC 8265, if that's the one in your S510UN.

**Comment by LeeBinder** [comment]
Yep, I'm already 90% done here on my VivoHack. Should be possible hopefully by Sunday nite as long as no bombs drop :)

**Comment by LeeBinder** [comment]
I've already updated the repo on my Vivo but GH Desktop for Mac (latest version) is stuck idling. Already quit, relaunched; signed out, back in - no change.

Might be a hickup of their system. Have to catch some sleep now and can see Mo. nite if pushing an update works again.

**Comment by LeeBinder** [comment]
I just updated the repo, both OC and Clover. Please download the latest [zip](https://github.com/LeeBinder/Asus-Vivobook-S510UA-Hackintosh/archive/refs/heads/main.zip) and let me know how it went. Don't forget to input your own SMBIOS/ PlatformInfo ;)

**Comment by abhishekthomasv** [comment]
> I just updated the repo, both OC and Clover. Please download the latest [zip](https://github.com/LeeBinder/Asus-Vivobook-S510UA-Hackintosh/archive/refs/heads/main.zip) and let me know how it went. Don't forget to input your own SMBIOS/ PlatformInfo ;)

Thanks alot @LeeBinder

**Comment by LeeBinder** [comment]
@abhishekthomasv couple of questions to you:

1. are you running macOS Sonoma?
2. if so, which Wi-Fi card are you using: original Intel or replacement? If the latter, which one?
3. if Wi-Fi working for you in Sonoma?

@cool2apps what about you:

1. which macOS are you running with the latest repo zip here?
2. is sound working for you now?

**Comment by abhishekthomasv** [comment]
currently I'm on ventura.
Im using an intel wifi card.
Esrlier when i was on monterey airplay and mirroring to vivobook used to work fine. But now in ventura doesn't work. Trying to find a solution to that.

**Comment by LeeBinder** [comment]
gotcha. Sorry I cannot help with that issue at all. Try over at the Intel Wi-Fi GitHub pages. Good luck to you 👍

**Comment by abhishekthomasv** [comment]
> gotcha. Sorry I cannot help with that issue at all. Try over at the Intel Wi-Fi GitHub pages. Good luck to you 👍 

Hey btw, I just booted sonoma with Clover. Had to use itlwm along with heliport for intel(stock) wifi to work. To fix bluetooth I had to add the bluetooth kexts. Also ended up adding featureUnlock kext for handoff to work.

Once again thanks for the EFI
@LeeBinder

**Comment by abhishekthomasv** [comment]
Unfortunately on Sonoma sleep results in shutdown. Also Which version of mac OS do you use? I'm thinking of going back to something more stable.

**Comment by cool2apps** [comment]
> @abhishekthomasv couple of questions to you:
> 
>     1. are you running macOS Sonoma?
> 
>     2. if so, which Wi-Fi card are you using: original Intel or replacement? If the latter, which one?
> 
>     3. if Wi-Fi working for you in Sonoma?
> 
> 
> @cool2apps what about you:
> 
>     1. which macOS are you running with the latest repo zip here?
> 
>     2. is sound working for you now?

Sorry for late reply. I am using Ventura. But I am not using macOS daily, just when building iOS apps. So I haven't used it for a while. When I will use, I'll report what is working what is not working. Thanks.

**Comment by LeeBinder** [comment]
@abhishekthomasv my daily driver is still Monterey because I regularly use Paragon Partition Manager which stopped working with Ventura. I also had Ventura installed and updated it to Sonoma. Sleep and wake up are/ have been working fine in all OSes and also regardless if booted via OC or Clover.

Have you tried all steps from the [Troubleshooting](https://github.com/LeeBinder/Asus-Vivobook-S510UA-Hackintosh#troubleshooting) section?

If so, disable all Wi-Fi kexts, reboot, test sleep and wake again.
